UPGRADE.md

```diff
@@ -0,0 +1,139 @@
+### Kubevirt HCO upgrade procedure
+
+Please note that the instructions were tested w/ OpenShift 4.2.
```

Comment:
The instructions should work regardless of OCP release. I don't think you need to include this note.

Reply:
The note was there to make sure nobody tries it w/ OpenShift version <4.2, i.e. 4.1.
But I agree it's mostly irrelevant now.
Removing.